package.json

```diff
@@ -31,7 +32,9 @@
   "devDependencies": {
     "babel-eslint": "^10.0.1",
     "eslint": "^5.16.0",
-    "eslint-plugin-html": "^5.0.5"
+    "eslint-plugin-html": "^5.0.5",
```

Comment:
This should probably be removed. You can remove the html plugin from your eslint config to get rid of this dependency.

Reply:
Looks like the html plugin isn't in the config file, and this is just hanging around from before.